docs/tasks/inject-data-application/podpreset-replicaset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
please consider remove the comment because it is confusing and inaccurate

Reply:
Sure, removed the comment. @tengqm